src/main/resources/application.properties

```diff
@@ -6,7 +6,7 @@ spring.servlet.multipart.enabled=true
 spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 #postgres
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-spring.datasource.url=jdbc:postgresql://${POSTGRES_URL:localhost:5432}/postgres
```

Comment:
please change back to the default port 5432, as this may result in additional setup for the other developers